package/MDAnalysis/core/groups.py

```diff
@@ -1182,17 +1214,63 @@ def select_atoms(self, sel, *othersel, **selgroups):
                 therefore have atoms that were initially absent from the
                 instance :meth:`~select_atoms` was called from.
 
+            global *selection*
```

Comment:
Sorry, my previous comment was totally off (mistook the `global` keyword for `group`). As you said, `selection` is just the placeholder. I'll leave it as `*selection*`.

Incidentally, the `group *group-name*` docs should be the ones changed to backticks, since `group-name` refers to a kwarg name, right?